### DISCUSSION TITLE
Help how do I get the YOLO badge?

### DISCUSSION THREAD

**Discussion by Binardo12** [community/community#19675] [Other Feature Feedback, Questions, & Ideas] [answered]
Help how do I get the YOLO badge?

**Comment by Nik-08** [comment, marked as answer]
Working way. You just need to:

Open your repository;
Create new branch;
Do some updates;
Then, create pull request to main branch;
IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;
After this, just click merge as you do it always
Check your profile and see YOLO achievement.
